frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -132,17 +132,15 @@ describe("scenarios > collection_defaults", () => {
         //
       });
 
-      it.skip("should expand/collapse collection tree by clicking on parent collection name (metabse#17339)", () => {
+      it("should expand/collapse collection tree by clicking on chevron)", () => {
```

Comment:
Was there a specific reason why you removed the issue reference from the test title?

Reply:
Also, "on parent collection name" was intentional. It's not just the chevron that triggers the action. The title is now a bit misleading.

Reply:
Did you see it here ? https://github.com/metabase/metabase/pull/21489#issuecomment-1090613496 

As I understood, the behavior changed. 

And I rename it following https://github.com/metabase/metabase/pull/21489#issuecomment-1090666673

Reply:
I missed that comment previously.
Just opened the local dev and I don't see anything changed. When I click on the parent collection name, it still opens and collapses.

Reply:
NP. 

When you click on the collection it opens, but to collapse you need to click twice. One click is to select only.